spectral_cube/spectral_cube.py

```diff
@@ -791,7 +791,7 @@ def apply_function(self, function, axis=None, weights=None, unit=None,
             the cube.
         update_function : function
             An alternative tracker for the progress of applying the function
-            to the cube data. If `progressbar` is `True`, this argument is
+            to the cube data. If progressbar is true, this argument is
```

Comment:
You don't like using double-backticks?  I realize my comment before just looked like ``progressbar``; I was trying to show \`\`progressbar\`\`

Reply:
Hah, got it! I was wondering if that's what you meant, but figured there was some sphinx flag that just wouldn't allow formatting in docstrings for whatever reason.

Reply:
Yeah, I think this should work now.  Let's see....